nucleus/admin/rest/rest-service/src/main/java/org/glassfish/admin/rest/resources/CollectionLeafResource.java

```diff
@@ -258,16 +260,8 @@ protected ActionReportResult buildActionReportResult() {
     
     private Map<String, String> optionToMap(JvmOption option){
         Map baseMap = new HashMap<>();
```

Comment:
Could this baseMap be a Map<String, String>?

Reply:
Yes, thanks. It's not pedantic, it's right thing to do, but I didn't read that line while fixing :)

Reply:
Thank you.